presentation_powerpoint.tex

```diff
@@ -151,13 +151,35 @@ \section{Discussion}
 \begin{frame}
 \frametitle{Other approaches}
 
+\textbf{Other classification methods}
+
+\begin{itemize}
+    \item One limitation of a classification tree is that it is very sensitive to small changes of a variable of interest.
+    \item Another option for this supervised classification problem could be to apply a robust logistic regression.
+    \item The main advantage of the tree over this regression method is that it can easily deal with outliers and does not require normalization and scaling of data.
+\end{itemize}
+
+\textbf{Deep Learning}
+
+\begin{itemize}
+    \item A master's thesis was written which attempted to apply deep learning to calculate CLV with a P$\&$C insurance company
+    \item 
+    \item (link to reference)
```

Comment:
Link to the paper: https://www.diva-portal.org/smash/get/diva2:1431621/FULLTEXT02.pdf